openquake/hazardlib/site.py

```diff
@@ -252,6 +253,7 @@ def from_points(cls, lons, lats, depths=None, sitemodel=None,
             self._set('z2pt5', sitemodel.reference_depth_to_2pt5km_per_sec)
             self._set('siteclass', sitemodel.reference_siteclass)
             self._set('backarc', sitemodel.reference_backarc)
+            self._set('soiltype', sitemodel.reference_soiltype)
```

Comment:
Please remove this line and the test should become green again

Reply:
But then how can we define a reference soiltype for all the sites in a region?

Reply:
Just write a site model with soiltype constant for all sites. If you look at the PR that I mentioned before (https://github.com/gem/oq-engine/pull/6672/files) the "bas" term is introduced only as a local site parameter. Not introducing `reference_soiltype` not only makes your life easier, it also avoid building a site collection larger than needed for all models not using Idini2017, i.e. all model in existence before this PR ;-)
It was a mistake to introduce the siteclass as a global parameter, I should remove that one too.

Reply:
oh, you are totally right! let me remove the line update